packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -57,6 +67,9 @@ class UnThemedLogRow extends PureComponent<Props, State> {
   };
 
   toggleDetails = () => {
+    if (this.props.isLogsPanel) {
```

Comment:
Nitpicking here and not relevant to this PR, but it would probably make more sense to have `allowDetails` as a prop that is set true/false depending on context. That way the information is more about what capabilities this component should provide which is more relevant here, instead of information about particular context (which this component should not care about) which then needs to be translated to specific capabilities anyway.

Reply:
Makes sense. I'll use `allowDetails` instead of `isLogsPanel`.